app/views/api/users/index.xml.builder

```diff
@@ -1,4 +1,4 @@
 xml.instruct! :xml, :version => "1.0"
 xml.osm(OSM::API.new.xml_root_attributes) do |osm|
-  osm << render(@users)
+  osm << (render(@users) || "")
```

Comment:
Is this actually necessary? Won't an empty set render sensibly? If it is needed that what about the JSON version?

Reply:
I think so, see also https://github.com/openstreetmap/openstreetmap-website/pull/2223#discussion_r300065590
If the set is empty, rendering fails with `ActionView::Template::Error: no implicit conversion of nil into String`, because `render()` returns nil in that case and trying to concatenate nil with an existing string will fail (but concatenating an empty string with another string works).
Regarding the JSON version, this is not an issue because no strings are concatenated there.